### PR TITLE
[REF][stock_move_entries] inherit _prepare_account_move_line for sm_id assign

### PR DESCRIPTION
**Pull request by ernesto-gm**
[REF][stock_move_entries] inherit _prepare_account_move_line for sm_id assign

### CODE REVIEW

File: stock_move_entries/model/stock_move_entries.py [hunk 1]
```diff
@@ -24,128 +25,40 @@
...
-                res[record_id] = sm_id
-            else:
-                res[record_id] = False
-        return res
-
     _columns = {
-        'sm_id': fields.function(_get_sm, method=True, type='many2one',
-                                 relation='stock.move',
-                                 string='Stock move ID', store=True),
+        'am_id': fields.one2many(
```
Comment (hbto) [inline stock_move_entries/model/stock_move_entries.py]: Let us change the name of this field from am_id to aml_ids,
to adhere to established convention of using _ids for X2many fields.

File: stock_picking_show_entries_info/model/account_move_line.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    _inherit = "stock.picking"
+
+    def show_entry_lines(self, cr, uid, ids, context=None):
+        ids = isinstance(ids, (int, long)) and [ids] or ids
+        context = context or {}
+        res = []
+        for picking_brw in self.browse(cr, uid, ids, context=context):
+            res += [
+                aml_brw.id for move in picking_brw.move_lines if move.am_id
+                for aml_brw in move.am_id.line_id]
```
Comment (hbto) [inline stock_picking_show_entries_info/model/account_move_line.py]: As we are suggesting changing the name from am_id to aml_ids, hence this variable shall be updated.

**Comment by ernesto-gm** [comment]
@hbto realice las correcciones que me comentaste

**Comment by hbto** [comment]
+1

**Pull request merged**